service_configuration_lib/spark_config.py

```diff
@@ -1184,11 +1184,10 @@ def compute_requested_memory_overhead(spark_opts: Mapping[str, str], executor_me
 
 def get_grafana_url(spark_conf: Mapping[str, str]) -> str:
     return (
-        'https://grafana.yelpcorp.com/d/b8f79180-bea8-4001-9d4a-94978b5a20b6/spark-on-paasta-job-status?orgId=1&'
+        'http://y/spark-monitoring?'
```

Comment:
This actually works? 
That's super amazing. TIL

Reply:
yeah it's cool! I tested the link and it worked